src/lib.rs

```diff
@@ -174,6 +176,35 @@ pub fn broadcast<T>(cap: usize) -> (Sender<T>, Receiver<T>) {
 /// separate semaphore to manage such limits does not work with broadcast channels as only the
 /// channel tracks when the last copy of a message is removed.  See [`CapacityLimiter`] for more
 /// details.
+///
+/// # Examples
+///
+/// ```
+/// # futures_lite::future::block_on(async {
+/// use async_broadcast::*;
+///
+/// let limit = BoxlikeCapacityLimit { limit: 199 };
+/// let (send, mut recv) = broadcast_with_limiter(limit);
+///
+/// let item = vec![0u8;100];
+/// send.try_broadcast(item).unwrap();
+///
+/// let item = vec![0u8;100];
+/// let fail = send.try_broadcast(item);
+/// assert!(matches!(fail, Err(TrySendError::Full { .. })));
+///
+/// let smol = vec![0u8;10];
```

Comment:
`smol`?

Reply:
A small tribute to the name of the project.